net-p2p/deluge/deluge-2.0.3.ebuild

```diff
@@ -0,0 +1,132 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{5,6} )
+DISTUTILS_SINGLE_IMPL=1
+inherit distutils-r1 eutils systemd user
+
+DESCRIPTION="BitTorrent client with a client/server model"
+HOMEPAGE="https://deluge-torrent.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.deluge-torrent.org/deluge"
+	SRC_URI=""
+else
+	SRC_URI="https://ftp.osuosl.org/pub/deluge/source/2.0/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3-with-openssl-exception"
+SLOT="0"
+IUSE="console geoip gtk libnotify sound webinterface"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	sound? ( gtk )
+	libnotify? ( gtk )
+"
+
+DEPEND="net-libs/libtorrent-rasterbar[python,${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	|| ( dev-python/slimit[${PYTHON_USEDEP}]
+		dev-lang/closure-compiler-bin )
+	dev-python/wheel[${PYTHON_USEDEP}]
+	dev-util/intltool"
+RDEPEND="net-libs/libtorrent-rasterbar[python,${PYTHON_USEDEP}]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/distro[${PYTHON_USEDEP}]
+	dev-python/rencode[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]
+	dev-python/pyopenssl[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	dev-python/setproctitle[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	>=dev-python/twisted-17.1[${PYTHON_USEDEP}]
+	x11-misc/xdg-utils
+	geoip? ( dev-python/geoip-python[${PYTHON_USEDEP}] )
+	gtk? (
+		dev-libs/libappindicator:3[introspection]
+		dev-python/pygobject:3[cairo,${PYTHON_USEDEP}]
+		gnome-base/librsvg
+		libnotify? ( x11-libs/libnotify[introspection] )
+		sound? ( dev-python/pygame[${PYTHON_USEDEP}] )
+	)
+	webinterface? ( dev-python/mako[${PYTHON_USEDEP}] )"
+
+python_prepare_all() {
+	local args=(
+		-e "/Compiling po file/a \\\ \\ \\ \\ \\ \\ \\ \\ upto_date = False"
+	)
+	sed -i "${args[@]}" -- 'setup.py' || die
+	args=(
+		-e "s|'new_release_check': True|'new_release_check': False|"
+	)
+	sed -i "${args[@]}" -- 'deluge/core/preferencesmanager.py' || die
+
+	distutils-r1_python_prepare_all
+}
+
+esetup.py() {
```

Comment:
Overriding eclass functions is not acceptable.

Reply:
Ok. I understand.
Do you have any suggestion on how to achieve the same result. I have no idea what to do.

Reply:
Maybe try overriding whatever you need to override via setup.cfg.